docs/changelog/0.5.0.draft.md

```diff
@@ -147,7 +148,7 @@ import scala.scalanative.posix.sys.stat
 import scala.scalanative.posix.timeOps._
 import scala.scalanative.posix.sys.statOps.statOps
 
-Zone { implicit z =>
+Zone {
```

Comment:
Is the lack of a `.apply()` and the dropped `{implicit z =>`
intentional?  In a couple of hundred other places, that seems to be the "Support both Scala 2 & 3" idiom.

If this one is different by intent, a comment would educate people
why a different idiom was used.  Looks like code which wants
to be Scala 2 & 3.